Foxtrot/Driver/Rewriting/EmitAsyncClosure.cs

```diff
@@ -59,6 +58,121 @@ namespace Microsoft.Contracts.Foxtrot
     /// </remarks>
     internal class EmitAsyncClosure : StandardVisitor
     {
+        /// <summary>
+        /// Class that maps generic arguments of the enclosed class/method the generic arguments of the closure.
+        /// </summary>
+        /// <remarks>
+        /// The problem.
+        /// Original implementation of the Code Contract didn't support async postconditions in generics.
+        /// Here is why:
+        /// Suppose we have following function (in class <code>Foo</code>:
+        /// <code><![CDATA[
+        /// public static Task<T> FooAsync() where T: class
+        /// {
+        ///     Contract.Ensures(Contract.Result<T>() != null);
+        /// }
+        /// ]]></code>
+        /// In this case, ccrewrite will generate async closure class called <code>Foo.AsyncContractClosure_0&lt;T%gt;</T></code> 
```

Comment:
Should be:
In this case, ccrewrite will generate async closure class called \<**c**\>Foo.AsyncContractClosure_0\&lt;T**&**gt;\</**c**\>